docs/coverage-reporter/alternative-ways-of-running-coverage-reporter.md

```diff
@@ -0,0 +1,172 @@
+---
+description: There are alternative ways of running or installing Codacy Coverage Reporter, such as running a Docker image, using a GitHub Action or CircleCI orb, downloading a binary for your operating system, or building the binary from source.
+---
+
+# Alternative ways of running Coverage Reporter
+
+The following sections list the alternative ways of running or installing Codacy Coverage Reporter.
+
+## Bash script (recommended) {: id="bash-script"}
+
+The recommended way to run the Codacy Coverage Reporter is by using the [self-contained bash script `get.sh`](https://github.com/codacy/codacy-coverage-reporter/blob/master/get.sh) that automatically downloads and runs the most recent version of the Codacy Coverage Reporter:
+
+-   On Ubuntu, run:
+
+    ```bash
+    bash <(curl -Ls https://coverage.codacy.com/get.sh)
+    ```
+
+-   On Alpine Linux, run:
+
+    ```sh
+    wget -qO - https://coverage.codacy.com/get.sh | sh
+    ```
+
+!!! note
+    Starting on version `13.0.0` the script automatically validates the checksum of the downloaded binary. To skip the checksum validation, define the following environment variable:
+
+    ```bash
+    export CODACY_REPORTER_SKIP_CHECKSUM=true
+    ```
+
+The self-contained script can cache the binary. To avoid downloading the binary every time that the script runs, add one of the following directories to your CI cached folders:
+
+-   `$HOME/.cache/codacy` on Linux
+-   `$HOME/Library/Caches/Codacy` on Mac OS X
+
+To use a specific version of the Codacy Coverage Reporter, set the following environment variable to one of the [released versions](https://github.com/codacy/codacy-coverage-reporter/releases):
+
+```bash
+export CODACY_REPORTER_VERSION=<version>
+```
+
+## Docker
+
+You can use Docker to run Codacy Coverage Reporter.
+
+Use the following command where `<version>` is either one of the [released versions](https://github.com/codacy/codacy-coverage-reporter/releases), or `latest` to use the most recent version:
+
+```bash
+docker run -v $PWD:/code codacy/codacy-coverage-reporter:<version> report
+```
+
+## GitHub Action
+
+If you're using GitHub Actions to report coverage, you can use our GitHub Action [codacy/codacy-coverage-reporter-action](https://github.com/codacy/codacy-coverage-reporter-action).
```

Comment:
The names of repositories usually trigger false positives from Vale because they include "non-dictionary" words, namely `codacy` (in lowercase).

You can tell Vale to ignore specific words using a span tag with a specific class, like this:

```suggestion
If you're using GitHub Actions to report coverage, you can use our GitHub Action [<span class="skip-vale">codacy/codacy-coverage-reporter-action</span>](https://github.com/codacy/codacy-coverage-reporter-action).
```

You can also check other places in the existing docs where we use this syntax.